exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "A basket containing only a single book is charged full price.",
+            "basket": [1],
+            "expected": 8.0
+         },
+         {
+	    "description": "A basket containing only two of the same book is charged full price.",
+            "basket": [2, 2],
+            "expected": 16.0
+         },
+         {
+            "description": "No charge to carry around an empty basket.",
+            "basket": [],
+            "expected": 0.0
+         },
+         {
+	    "description": [
+		             "A basket containing two different books is the cheapest grouping, ",
+		             "better than purchasing each book on its own."
+		           ],
+            "basket": [1, 2],
+            "expected": 15.2
+         },
+         {
+	    "description": "A basket of three different books is cheaper than grouping the books any other way.",
+            "basket": [1, 2, 3],
+            "expected": 21.60
+         },
+         {
+	    "description": "A basket of four different books is cheaper than grouping the books any other way.",
+            "basket": [1, 2, 3, 4],
+            "expected": 25.60
+         },
+         {
+	    "description": "A basket of five different books is cheaper than grouping the books any other way.",
+            "basket": [1, 2, 3, 4, 5],
+            "expected": 30.00
+         },
+         {
+	    "description": [
+		             "A basket containing three pairs of three different books plus ",
+		             "one copy each of the other two books, not paired, is the least expensive ",
+		             "grouping."
+		           ],
+            "basket": [1, 1, 2, 2, 3, 3, 4, 5],
+            "expected": 51.20
+         },
+         {
+	    "description": [
+		             "A basket containing four pairs of four different books plus ",
+		             "one copy of a book, not already paired, is the least expensive grouping."
```

Comment:
@petertseng thanks for the feedback.  Spacing issue should be resolved.  I was totally missing that the descriptions should contain the solution that yields the expected result, duh.  I have re-written the descriptions.  I left them wordy.  If you think being too wordy is harder to follow I could, for example, rewrite to be like given basket [1,1,2,2,3,3,4,4,5] group as [1,2,3,4,5] and [1,2,3,4]....  Let me know what you think.